bot/tasks/pennychat.py

```diff
@@ -240,6 +242,45 @@ def _penny_chat_details_blocks(penny_chat_invitation, mode=None):
         date_time_block
     ]
 
+    if penny_chat_invitation.video_conference_link:
+        body.append(
+            {
+                'type': 'section',
+                'text': {
+                    'type': 'mrkdwn',
+                    'text': f'*Video Call Link*'
```

Comment:
Nit: If there is no formatting, then don't use an f-string. It probably doesn't matter much, but I bet there is an extra parsing step that happens. (below as well)

Reply:
Yeah flake caught this and I fixed in my most recent commit